tools/gen_allocs.c

```diff
@@ -589,30 +589,97 @@ gen_dealloc1 ( char * dirname )
   FILE * fp ;
   char  fname[NAMELEN] ;
   char * fn = "deallocs.inc" ;
+  // Open array of deallocs_[n].inc
+  int   numFiles = 12;
+  int    idx      = 0;
+  FILE * fpSub; 
+  char * filename_prefix = "deallocs_" ;
```

Comment:
It might be nice to use consistent whitespace here between the variable type, variable name, `=` character, and trailing semicolon.